crates/lib/src/install.rs

```diff
@@ -90,7 +90,7 @@ pub(crate) const ARCH_USES_EFI: bool = cfg!(any(target_arch = "x86_64", target_a
 #[cfg(any(feature = "composefs-backend", feature = "install-to-disk"))]
 pub(crate) const ESP_GUID: &str = "C12A7328-F81F-11D2-BA4B-00A0C93EC93B";
 #[cfg(any(feature = "composefs-backend", feature = "install-to-disk"))]
-pub(crate) const DPS_UUID: &str = "6523f8ae-3eb1-4e2a-a05a-18b695ae656f";
+pub(crate) const DPS_UUID: &str = "4f68bce3-e8cd-4db1-96e7-fbcaf984b709";
```

Comment:
This is switching to hardcode x86_64 note, what we actually need is to do this per-arch properly